src/main/java/net/dv8tion/jda/internal/requests/WebSocketClient.java

```diff
@@ -1106,6 +1122,25 @@ else if (requestStage == ConnectionStage.CONNECT)
         return null;
     }
 
+    public void requestMembers(Consumer<DataObject> callback, long guildId, long userId)
```

Comment:
Should we support requesting multiple in one request so that users don't have to make a ton of requests to to the work that a single could provide?

Reply:
I'm not sure if its worth supporting that. If you need access to multiple presences its likely better to just enable the cache. The interface only becomes more complicated.

Reply:
Not sure I agree with this all or single mentality. Seems perfectly reasonable to want to request more than one at a timeout without wanting the entire cache.